### ISSUE TITLE
Unable to create Google Maps custom infoWindow on iOS 

### ISSUE THREAD

**Issue by dlcole** [NativeScript/plugins#564]
This code works on Android to create a custom infoWindow using the `@nativescript/google-maps` plugin: 

```
export function onMarkerInfoWindow(args) {
    const marker = args.marker;
    const info = new StackLayout();
    info.width = 200;
    info.height = 100;
    info.backgroundColor = 'white';
    const text = new Label();
    text.text = marker.title;
    info.addChild(text);
    args.view = info;  
} 
```

On iOS, however, the default infoWindow is displayed.  

Here's the [code in index.ios.ts](https://github.com/NativeScript/plugins/blob/1a1df709f2313d5f45796823941914392b373e26/packages/google-maps/index.ios.ts#L402C2-L415C3) that raises the `MarkerInfoWindow` event. 

The problem is that `event.view.ios` is null, and that causes the default infoWindow to be used.  The native view needs to be created for the Stackayout returned by `onMarkerInfoWindow`.

This issue is also discussed in this [Discord thread](https://discord.com/channels/603595811204366337/1196261322027966484).

**Comment by dlcole** [comment]
I have created a [GitHub repo](https://github.com/dlcole/InfoWindowTester) to demo the problem.  I modified the code here to catch the `markerInfoContents` event rather than `markerInfoWindow`, as this retains the pointer at the bottom of the infoWindow.

Using the new 1.7.1 version of @nativescript/google-maps, I can now at least see a small infoWindow on iOS, but it contains no data.  See the screen shots, below... 
     
![Screenshot_1706236711](https://github.com/NativeScript/plugins/assets/11671244/0196b707-6219-4c10-a3b0-ae4f2b438f3b)
![Simulator Screenshot - iPhone 15 Pro - 2024-01-25 at 21 38 03](https://github.com/NativeScript/plugins/assets/11671244/25caf7fa-8c4c-4861-9d7f-3b34f52e711b)
